scripts/protein/esm2/esm2_pretrain.py

```diff
@@ -247,6 +249,7 @@ def main(
         save_top_k=save_top_k,
         every_n_train_steps=val_check_interval,
         always_save_context=True,  # Enables the .nemo file-like checkpointing where all IOMixins are under SerDe
+        filename='{epoch}-{val_loss:.2f}-{step}-{consumed_samples}'
```

Comment:
Add a comment here saying that `step` and `consumed_samples` are included to make sure names are unique.